src/Gui/NavigationStyle.h

```diff
@@ -415,6 +415,9 @@ class GuiExport TouchpadNavigationStyle : public UserNavigationStyle {
 
 protected:
     SbBool processSoEvent(const SoEvent * const ev) override;
+
+private:
+    SbBool blockPan {false}; // Used to block the first pan in a mouse movement to prevent big jumps

```

Comment:
Let's consider making this a static local since it's only used in one method.

Reply:
Making it static would make it global wrt. lifetime - initialized on program start and shared among all instances of the class. I'd not go there. For me it is harder to reason about.

Reply:
Done

Reply:
Oh I did not see that second message, I don't really mind but I think @zbynekwinkler has a good point.

Reply:
I'm fine with either way, I'm always just a little cautious about maintaining a piece of state like this: as a static local it's guaranteed that no other method is touching it. This method is long enough that I agree with @zbynekwinkler that it's not a sure thing that a static would be an improvement, so if you'd rather not, I'm fine with that.

Reply:
Then I am not making it static.